routing/turns_notification_manager.cpp

```diff
@@ -176,7 +179,8 @@ string NotificationManager::GenerateFirstTurnSound(TurnItem const & turn,
           m_nextTurnNotificationProgress = PronouncedNotification::First;
           return GenerateTurnText(roundedDistToPronounceUnits, turn.m_exitNum,
                                   false /* useThenInsteadOfDistance */, turn,
-                                  m_settings.GetLengthUnits());
+                                  m_settings.GetLengthUnits(),
```

Comment:
Here and in other places, it should fit in one line. Fewer lines of code = easier to read it. We _try_ to not exceed 120 chars, but it is allowed sometimes to improve readability.

Reply:
I'm making the minimal change possible here; this function invocation is often written this way because of all the /* comments */ in and among these calls explaining why they're not all the same. I could redo all invocations but there's not really a great reason for me to do so, I tried and it doesn't improve readability that much.